node_modules/gh-users/lib/api.js

```diff
@@ -272,6 +272,48 @@ var updateUser = module.exports.updateUser = function(ctx, userId, update, callb
     });
 };
 
+/**
+ * Update the app administrator status for a user
+ *
+ * @param  {Context}        ctx                             The current execution context containing the user and application
```

Comment:
Can you consolidate the `ctx` descriptions in this file with the other `ctx` descriptions in the application?

```
Standard context object containing the current user and the current application
```